File: assets/samples/dns.lookup/php/dns_lookup_apikey.php
<?php

$domain = 'example.com';
$apiKey = 'Your DNS Lookup API Key';
$type = '_all';

$url = 'https://www.whoisxmlapi.com/whoisserver/DNSService?'
    . "apiKey={$apiKey}&domainName={$domain}&type={$type}";

print(file_get_contents($url));

File: assets/samples/dns.lookup/python/dns_lookup_apikey.py
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

domain = 'example.com'
apiKey = 'Your DNS Lookup API Key'
type = '_all'

url = 'https://www.whoisxmlapi.com/whoisserver/DNSService?'
    + 'apiKey=' + apiKey + '&domainName=' + domain + '&type=' + type

print(urlopen(url).read().decode('utf8'))
